components/eam/src/chemistry/modal_aero/modal_aero_drydep.F90

```diff
@@ -271,12 +273,13 @@ subroutine aero_model_drydep  ( state, pbuf, obklen, ustar, cam_in, dt, cam_out,
 
           call outfld( trim(cnst_name(icnst))//'DDV', vlc_dry(:ncol,:,jvlc), pcols, lchnk )
 
-      enddo ! lspec = 1, nspec_amode(m)
-    enddo   ! imode = 1, ntot_amode
+       enddo ! lspec = 1, nspec_amode(m)
+    enddo    ! imode = 1, ntot_amode
 
     !=====================================================================
-    ! if the user has specified prescribed aerosol dep fluxes then 
-    ! do not set cam_out dep fluxes according to the prognostic aerosols
+    ! Unless the user has specified prescribed aerosol dep fluxes,
+    ! copy the fluxes calculated here to cam_out to be passed to other 
+    ! components of the Earth System Model.
     !=====================================================================
     if (.not.aerodep_flx_prescribed()) then
```

Comment:
I think this if block is always true for standard run. Only single column run uses prescribed. We may remove the if condition. Also, we will need to refactor subroutine set_srf_drydep for the next step. It groups depositon fluxes and passes to coupler.

Reply:
The if-statement is removed in [0b3942c](https://github.com/eagles-project/e3sm_mam4_refactor/pull/68/commits/0b3942cbb90d34a6a933a0b5f2a9ca043ea649c2)